packages/core/organisms/PersianCalendar/PersianCalendar.vue

```diff
@@ -90,6 +167,160 @@ const selectingDate = (day: number) => {
         </template>
       </div>
     </div>
+    <div v-else class="flex flex-col items-center mb-4 px-4">
+      <div class="flex items-center justify-between mb-4 w-[100%] mt-3">
+        <div>
+          <div v-if="steps.year">
+            <slot name="previous-year-page-icon" :previous-page="prevPage">
+              <PreviousYearArrow
+                @click="prevPage"
+                :disabled="currentPage === 0"
+                class="bg-blue-500 border-none rounded-full !w-[12px] !h-[12px] text-white p-2 fill-white mx-1 hover:bg-blue-200 hover:text-blue-600 cursor-pointer transition-all"
+              />
+            </slot>
+          </div>
+          <div v-if="steps.month">
+            <slot name="return-icon" :month-step="toYearStep">
+              <ReturnArrow
+                class="bg-blue-500 border-none rounded-full !w-[12px] !h-[12px] text-white p-2 fill-white mx-1 hover:bg-blue-200 hover:text-blue-600 cursor-pointer transition-all"
+                @click="toYearStep"
+              />
+            </slot>
+          </div>
+          <div v-if="steps.day">
+            <div class="flex justify-center items-center">
+              <slot name="previous-year-icon" :previous-year="previousYear">
+                <PreviousYearArrow
+                  @click="previousYear"
+                  :next-year="previousYear"
+                  class="bg-blue-500 border-none rounded-full !w-[12px] !h-[12px] text-white p-2 fill-white mx-1 hover:bg-blue-200 hover:text-blue-600 cursor-pointer transition-all"
+                />
+              </slot>
+              <slot name="previous-month-icon" :previous-month="previousMonth">
+                <PreviousMonthArrow
+                  @click="previousMonth"
+                  :next-month="previousMonth"
+                  class="bg-blue-500 border-none rounded-full !w-[12px] !h-[12px] text-white p-2 fill-white mx-1 hover:bg-blue-200 hover:text-blue-600 cursor-pointer transition-all"
+                />
+              </slot>
+            </div>
+          </div>
+        </div>
+        <div class="w-max mx-3 text-center" :style="{ color: 'black !important' }">
+          <div v-if="steps.year">
+            <PersianTypography
+              :text="`${yearPeriodRange[yearPeriodRange.length - 1]} - ${yearPeriodRange[0]}`"
+            />
+          </div>
+          <div v-else-if="steps.month" class="cursor-pointer">
+            <PersianTypography :text="String(currentYear)" />
+          </div>
+          <div v-if="steps.day" class="flex justify-center">
+            <a class="mx-1 cursor-pointer" :style="{ color: 'black' }" :onclick="monthStepShortCut">
+              <PersianTypography :text="monthName.split(' ')[0]" />
```

Comment:
```suggestion
              <PersianTypography :text="currentMonthName" />
```
and a computed like `const curerntMonthName = computed(() => monthName.value.split(' ')[0])`

Reply:
Property 'split' does not exist on type 'ComputedRef<string>'